narayana-spring-boot-starter/src/main/java/dev/snowdrop/boot/narayana/autoconfigure/NarayanaAutoConfiguration.java

```diff
@@ -67,12 +66,11 @@
         com.arjuna.ats.jta.UserTransaction.class
 })
 @ConditionalOnMissingBean(PlatformTransactionManager.class)
-@AutoConfigureBefore(JtaAutoConfiguration.class)
```

Comment:
In case it is useful to consider, I think the AutoConfigureBefore can be tracked into https://github.com/snowdrop/narayana-spring-boot/commit/5805c73ad3318232c086fb8ae2491fd17a7797d4